uPortal-webapp/src/main/webapp/WEB-INF/jsp/Search/searchLauncher.jsp

```diff
@@ -32,16 +32,17 @@
 <div id="webSearchContainer" class="fl-widget">
     <div class="fl-widget-inner">
       <div class="fl-widget-content">
-        <c:set var="searchLabel"><spring:message code="search"/></c:set>
         <form class="form-search" role="form" method="POST" action="${searchLaunchUrl}" id="${n}webSearchForm">
           <div class="input-group">
             <spring:message code="search" var="searchPlaceholder" />
             <spring:message code="search.terms" var="searchTitle" />
-            <spring:message code="submit" var="searchSubmit" />
-            <input id="${n}webSearchInput" class="searchInput input-large search-query form-control" value="" name="query" type="search" placeholder="${searchPlaceholder}" title="${searchTitle}"/>
+            <spring:message code="search.submit" var="searchSubmit" />
+            <input id="${n}webSearchInput" class="searchInput input-large search-query form-control" value="" name="query"
+             type="search" aria-label="${searchTitle}" placeholder="${searchPlaceholder}" title="${searchTitle}"/>
             <span class="input-group-btn">
-              <button id="webSearchSubmit" type="submit" name="submit" aria-label="${searchSubmit}" class="btn btn-default" value="${searchLabel}">
-                <span>${searchLabel}</span>
+              <button id="webSearchSubmit" type="submit" name="submit" aria-label="${searchSubmit}" class="btn btn-default"
+               value="${searchSubmit}">
+                <span>${searchSubmit}</span>
```

Comment:
if the text appears both in the content and in an `aria-label` it could be double read

```suggestion
              <button id="webSearchSubmit" type="submit" name="submit" class="btn btn-default"
               value="${searchSubmit}">
                <span>${searchSubmit}</span>
```